sklearn/kernel_approximation.py

```diff
@@ -245,9 +245,8 @@ class AdditiveChi2Sampler(BaseEstimator, TransformerMixin):
     References
     ----------
     See `"Efficient additive kernels via explicit feature maps"
-    <http://eprints.pascal-network.org/archive/00006964/01/vedaldi10.pdf>`_
-    Vedaldi, A. and Zisserman, A., Computer Vision and Pattern Recognition 2010
-
+    <http://www.robots.ox.ac.uk/%7Evedaldi/assets/pubs/vedaldi11efficient.pdf>`_
```

Comment:
Could you replace the `%7E` with a `~` please, and check that the link works in the HTML reference after rendering the docs? If this works, this is done. Thanks!